src/modding/ModManager.cs

```diff
@@ -509,7 +743,14 @@ private void RefreshEnabledMods()
 
         foreach (var mod in enabledMods)
         {
-            enabledModsContainer.AddItem(mod.InternalName, LoadModIcon(mod));
+            if (mod != null)
+            {
+                enabledModsContainer.AddItem(mod.InternalName, LoadModIcon(mod));
+                if (!string.IsNullOrEmpty(mod.Info.Description))
+                {
+                    SetModToolTip(enabledModsContainer, mod);
```

Comment:
Wouldn't it be fine to always run this code? As far as I remember description is mandatory field so it is at least a blank string. So it wouldn't hurt to display it, and plus it would work as a way to clear an existing description if the mod data was refreshed.